src/actions/game.js

```diff
@@ -0,0 +1,7 @@
+export const flick = light => light
+export const ready = () => {}
+export const reset = () => {}
+export const startTimer = () => {}
+export const stopTimer = () => {}
+export const tickTimer = () => {}
+export const toggleSettings = () => {}
```

Comment:
Some times I've wondered about just making these `toggleSettings = 'TOGGLE_SETTINGS'` vs the identify function. I still don't know.